projects/angular/src/wizard/_wizard.clarity.scss

```
@@ -246,14 +281,27 @@
     .clr-wizard-stepnav-item {
       display: block;
       // IE/OldEdge
-      box-shadow: wizard-variables.$clr-wizard-stepnav-border-size 0 0
-        wizard-variables.$clr-wizard-stepnav-item-border-color inset;
-      @if variables.$clr-use-custom-properties == true {
-        box-shadow: var(--clr-wizard-stepnav-border-size) 0 0 var(--clr-wizard-stepnav-item-border-color) inset;
-      }
+      border-left-style: solid;
+      @include mixins.css-var(
+        border-left-width,
+        clr-wizard-stepnav-border-size,
+        wizard-variables.$clr-wizard-stepnav-border-size,
+        variables.$clr-use-custom-properties
+      );
+      @include mixins.css-var(
+        border-color,
+        clr-wizard-stepnav-item-border-color,
+        wizard-variables.$clr-wizard-stepnav-item-border-color,
+        variables.$clr-use-custom-properties
+      );
+
       margin: 0 0 (-1 * variables.$clr-global-borderwidth) 0;
-      padding: variables.$clr_baselineRem_0_25 0;
-      padding-left: variables.$clr_baselineRem_8px;
+      @include mixins.css-var(
+        padding,
+        clr-wizard-stepnav-item-padding,
+        variables.$clr_baselineRem_0_25 0 variables.$clr_baselineRem_0_25 variables.$clr_baselineRem_4px,
```

Original  padding left is 8px `variables.$clr_baselineRem_8px;` not 4.

```suggestion
        variables.$clr_baselineRem_0_25 0 variables.$clr_baselineRem_0_25 variables.$clr_baselineRem_8px,
```

---

Originally the colored left "border" was made up with shadow and additional padding was needed. I change it to use border instead and that much padding is no longer needed.